### ISSUE TITLE
Parent Issue: Most Read as a user-facing URL

### ISSUE THREAD

**Issue by neildoughty** [bbc/simorgh#4472]
Currently all WS sites have a Most Read page as a URL users can visit. The URL is of the format:
`bbc.com/<service>/popular/read` eg. /igbo/popular/read

If we want to maintain this url, and I think at present we do, then we will need to recreate this page somewhere, for all services, and it seems making use of the new most read component in a page is logical.

**Describe the solution you'd like**
I would like a page in Simorgh to be available for each service of this format:
`bbc.com/<service>/popular/read`
If a user visits that page I would like them to be able to see a list of popular news stories from that service.
Accessibility - Most read should be within the main container and the h1 should be visually displayed as ‘Most read’ and the skip to content link should anchor here (note there should not be a h2 for most read or any other regions for most read within the main region) 

The outcome of this will be that we have a new page type and route in Simorgh, which will initially be available locally, and we can then get feedback from UX and make changes accordingly.

**Describe alternatives you've considered**
In theory, in a world in which we will be building other most popular lists of content there could be an argument that using the url `bbc.com/<service>/popular` might be better, since it would allow us to stack Most Read, Most Watched, Most popular topics etc in a single page.
But that would then require a bunch of redirects from /popular/read to /popular

**Testing notes**
To be tested according to design doc across various breakpoints.Also the ATI and chartbeat testing for this new page.

- Unit tests
- Integration tests
- One cypress test to cover this as it is a new page type.

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [x] This feature is expected to need manual testing.
**Additional context**
This piece of work is potentially a thing we could do, or not do. Before we get deep into it I'd be interested in an estimate of how much work is needed to make this happen for all sites.

**Comment by thekp** [comment]
I've done a small investigation which resulted in a small PR showcasing most read on a separate page (i.e bbc.com/pidgin/most/read).

From this PR, most read is fetching data on the client-side, this causes in the UI on load. Further investigation should be done to find out how we would fetch data for most read on the server. Parking this issue for the time being, until further notice.

**Comment by thekp** [comment]
**Accessibility notes:**

 h1 : `<h1 id="content" tabindex="-1">Most read</h1>`
main : `<main role="main">`

**Comment by thekp** [comment]
UX Design: 
![image](https://user-images.githubusercontent.com/30599794/78908541-24c96a80-7a7a-11ea-9bd6-0e23cd698697.png)
![image](https://user-images.githubusercontent.com/30599794/78908728-6b1ec980-7a7a-11ea-81ad-2bd239f13265.png)
![image](https://user-images.githubusercontent.com/30599794/78908711-62c68e80-7a7a-11ea-81c5-076677b73a82.png)

**Comment by thekp** [comment]
What's left: 
A11y swarm once Most read page is on TEST. Then we can create LIVE route after.

related issues/PRs:
https://github.com/bbc/simorgh/issues/6185
https://github.com/bbc/mozart-routing/pull/1301
https://github.com/bbc/mozart-smoke-test/pull/215

**Comment by thekp** [comment]
**Checklist for done:**
update getInitialData: https://github.com/bbc/simorgh/pull/6175 ✅ 
refactor most read: https://github.com/bbc/simorgh/pull/6171 ✅ 
add test route: https://github.com/bbc/mozart-routing/pull/1301 ✅ 
add smoke test: https://github.com/bbc/mozart-smoke-test/pull/215 ✅ 
a11y swarm: https://github.com/bbc/simorgh/issues/6185 ✅ 
metadata: https://github.com/bbc/simorgh/issues/6202  ✅ 
Fix Server logs: https://github.com/bbc/simorgh/issues/6362 ✅ 
AMP container for most read (using `amp-list`): https://github.com/bbc/simorgh/issues/6353 (blocked) 🚫 
Server-side render most read AMP page (`$service/popular/read.amp`): https://github.com/bbc/simorgh/pull/6441 ✅ 
add ".amp" rotues: https://github.com/bbc/mozart-routing/pull/1342 (merged) https://github.com/bbc/mozart-smoke-test/pull/235 ✅ 
add e2e (cypress) tests:https://github.com/bbc/simorgh/issues/6169 ✅ 
enhance error logs: https://github.com/bbc/simorgh/issues/6451 ✅ 
add bbc-a11y test: https://github.com/bbc/simorgh/issues/6394 ✅ 
Add to live route in mozart and add live smoke tests: https://github.com/bbc/simorgh-infrastructure/issues/981 ✅ 
Update Kyrgyz most read fixture data to include Optimo blocks - https://github.com/bbc/simorgh/issues/6500  ✅ 
Add most read page to lighthouse CI: https://github.com/bbc/simorgh-infrastructure/issues/1009 ✅ 

**Documentation:**
add pagemon dashboard for most read page: https://github.com/bbc/simorgh-infrastructure/issues/997 ✅ 
update runbook: https://github.com/bbc/simorgh-infrastructure/issues/996 ✅

**Comment by thekp** [comment]
The page is LIVE now and documentation has been updated
